spec/models/searchgov_url_spec.rb

```diff
@@ -380,6 +380,35 @@
       end
     end
 
+    context 'when the searchgov domain has js renderer enabled' do
+      let(:domain_with_js_renderer) do
+        instance_double(SearchgovDomain, check_status: '200 OK', js_renderer: true)
+      end
+
+      it 'fetches page with javascript response' do
```

Comment:
I think this test would pass for any HTML document, right?

Reply:
Yes, it would pass for any response that would start with `'<head>'`. Cannot determine the exact content type, here making sure JS content is rendered.